internal/controller/controlplane/k0s_controlplane_controller.go

```diff
@@ -437,13 +380,61 @@ func (c *K0sController) reconcileMachines(ctx context.Context, cluster *clusterv
 			return fmt.Errorf("waiting for previous machine to be deleted")
 		}
 
-		err := c.runMachineDeletionSequence(ctx, logger, cluster, kcp, machineToDelete)
+		err = c.runMachineDeletionSequence(ctx, logger, cluster, kcp, machineToDelete)
 		if err != nil {
 			return err
 		}
 
 		logger.Info("Deleted machine", "machine", machineToDelete.Name)
 	}
+
+	if len(desiredMachineNames) < int(kcp.Spec.Replicas) {
+
+		name := machineName(kcp, machineNamesToDelete, desiredMachineNames)
+		log.Log.Info("desire machine", "name", len(desiredMachineNames))
```

Comment:
```suggestion
		log.Log.Info("desire machine", "name", name)
```
Im not sure if the intention is this one but adding a suggestion